extensions/ext-tiled/src/main/java/org/pixel/ext/tiled/content/TiledConstants.java

```diff
@@ -0,0 +1,14 @@
+package org.pixel.ext.tiled.content;
+
+public enum TiledConstants {
+    PIXEL_EPSILON(0.005f);
```

Comment:
What should be private here? I can't make the enum package-private because it is used in the view package, but I can make the "value" property private

Reply:
The value only (the 's' was referring to other similar scenarios)